docs/index.md

```diff
@@ -18,6 +18,7 @@
     * As part of the text of a question
     * As part of the text of a default value
     * To decide if a question should be skipped
+    * To decide if a message should be displayed
```

Comment:
Minor: I think it would be cleaner if was included as part of the previous item instead of adding a new one.